index.html

```diff
@@ -37,261 +37,362 @@
-                    alt="framework icon"
-                    class="skill-icon"
-                  />
+              <div class="skill-wrapper card-secondary">
+                <div class="skill-card">
+                  <div class="skill-icon-container">
+                    <img
+                      src="./assets/icon-frameworks.svg"
+                      alt="framework icon"
+                      class="skill-icon"
+                    />
+                  </div>
+                </div>
+                <div class="skill-content skill-content">
+                  <h3 class="skill-title">Frameworks</h3>
+                  <ul class="skill-items">
+                    <li class="skill-item">React.js</li>
+                    <li class="skill-item">Ruby on Rails</li>
+                    <li class="skill-item">RSpec</li>
+                    <li class="skill-item">Capybara</li>
+                    <li class="skill-item">Selenium</li>
+                  </ul>
                 </div>
               </div>
-              <div class="skill-content skill-content">
-                <h3 class="skill-title">Frameworks</h3>
-                <ul class="skill-items">
-                  <li class="skill-item">React.js</li>
-                  <li class="skill-item">Ruby on Rails</li>
-                  <li class="skill-item">RSpec</li>
-                  <li class="skill-item">Capybara</li>
-                  <li class="skill-item">Selenium</li>
-                </ul>
+              <div class="skill-wrapper card-secondary">
+                <div class="skill-card">
+                  <div class="skill-icon-container">
+                    <img
+                      src="./assets/icon-skills.svg"
+                      alt="skill icon"
+                      class="skill-icon"
+                    />
+                  </div>
+                </div>
+                <div class="skill-content skill-content">
+                  <h3 class="skill-title">Skills</h3>
+                  <ul class="skill-items">
+                    <li class="skill-item">Database Management</li>
+                    <li class="skill-item">Version Control</li>
+                    <li class="skill-item">CLI</li>
+                    <li class="skill-item">Web Development</li>
+                    <li class="skill-item">API Design</li>
+                  </ul>
+                </div>
               </div>
             </div>
-            <div class="skill-wrapper card-secondary">
-              <div class="skill-card">
-                <div class="skill-icon-container">
+          </div>
+        </div>
+      </section>
+      <section id="contact" class="contact-background">
+        <div class="wrapper">
+          <div class="contact-wrapper">
+            <h2 class="contact-title">
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h2>
+            <form class="form">
```

Comment:
- [ ] Nice work implementing the form, it look really good :clap: However it states in the project requirement that data should be collected by using the [Formspree service](https://formspree.io/html). Therefore I suggest you:
  - add the `action` attribute to your form tag and its value should be similar to `https://formspree.io/f/{form_id}` where `{form_id}`  is `id` giving you when create an account on [formspree service](https://formspree.io/html).
  - add the `method` attribute to your `form` tag and its value  equals `POST`